site/com.jboss.devstudio.core.product

```diff
@@ -23,8 +23,6 @@ Red Hat, Inc. - Initial implementation.
    <launcherArgs>
       <programArgs>-product
 com.jboss.devstudio.core.product
--showsplash
```

Comment:
So will we get back to the upstream splash screen ? If yes then that's a no

Reply:
the splash path is configured in the config.ini file and correctly shown when starting eclipse. 
this is similar to https://bugs.eclipse.org/bugs/show_bug.cgi?id=575666.

Reply:
Not sure to get it: which config.ini file ? Isn't it generated from this information ?

Reply:
https://user-images.githubusercontent.com/600414/131706249-5e47f0fd-3636-4b8e-ad0a-e10214b7ea77.mov

here is my local build without the splash in ini file